libr/core/cmd_search.c

```diff
@@ -2349,7 +2353,11 @@ static int cmd_search(void *data, const char *input) {
                 //fin = ini + s->size;
         }
  */
-	maxhits = r_config_get_i (core->config, "search.maxhits");
+	core->search->maxhits = r_config_get_i (core->config, "search.maxhits");
+	// TODO default search.maxhits to UT64_MAX
+	if (!core->search->maxhits) {
```

Comment:
imho having 0 maxhits is more logic and easy to initialize than a value that is not zero. and it was already like this. also by the user it is easier to type e search.maxhits=0 than e search.maxhits=0xffffffffffffffffff